packages/api/pages/api/languages/[code]/words/[wordId]/gloss.ts

```diff
@@ -58,6 +66,18 @@ export default createRoute<{ code: string; wordId: string }>()
         },
       });
 
+      await client.glossHistory.create({
+        data: {
+          wordId: req.query.wordId,
+          languageId: language.id,
+          userId: req.session?.user?.id,
+          gloss:
+            originalGloss?.gloss !== fields.gloss ? fields.gloss : undefined,
```

Comment:
```suggestion
            fields.gloss !== originalGloss?.gloss ? fields.gloss : undefined,
```

"If new value doesn't equal the old value" is a bit more intuitive than "If old value doesn't equal new value". We're checking the new value in comparison to the old value, because the new value is the thing that might have changed.

Reply:
Done.